custom/icds_reports/static/js/directives/cas-export/cas-export.directive.js

```diff
@@ -51,19 +51,6 @@ function CasExportController($window, $location, locationHierarchy, locationsSer
         });
     }
     vm.years = vm.yearsCopy;
-    vm.filterOptions = [
-        {label: 'Data not Entered for weight (Unweighed)', id: 'unweighed'},
-        {label: 'Data not Entered for height (Unmeasured)', id: 'umeasured'},
-        {label: 'Severely Underweight', id: 'severely_underweight'},
-        {label: 'Moderately Underweight', id: 'moderately_underweight'},
-        {label: 'Normal (weight-for-age)', id: 'normal_wfa'},
-        {label: 'Severely Stunted', id: 'severely_stunted'},
-        {label: 'Moderately Stunted', id: 'moderately_stunted'},
-        {label: 'Normal (height-for-age)', id: 'normal_hfa'},
-        {label: 'Severely Wasted', id: 'severely_wasted'},
-        {label: 'Moderately Wasted', id: 'moderately_wasted'},
-        {label: 'Normal (weight-for-height)', id: 'normal_wfh'},
-    ];
```

Comment:
do you know when this became irrelevant?

Reply:
This was never relevant. It was clearly copied from `download.directive.js` when this file was created but is very specific to a single report on that other page (the child beneficiary list)

Reply:
oof. :disappointed: